### ISSUE TITLE
Cold starts fail following enkf atmos/mem swap

### ISSUE THREAD

**Issue by DavidHuber-NOAA** [NOAA-EMC/global-workflow#1275]
**Expected behavior**

The workflow scripts should be able to take the gdasenkf cold start ICs from gdas_init and organize them in the expected memxxx/atmos structure.

**Current behavior**

When running setup_expt.py with --icsdir specified, the script fails with a message in the form
`FileNotFoundError: [Errno 2] No such file or directory: '<ICS_DIR>/enkfgdas.YYYYMMDD/CC/mem001/atmos/INPUT'`

**Machines affected**

Hera at least.

**To Reproduce**

Check out a fresh copy of ufs_utils and generate ICs to a directory `$ICSDIR`
Run setup_expt.py with `--icsdir $ICSDIR`

Alternatively, run setup_expt.py and setup_workflow.py without `--icsdir` and start the experiment.  The efcs jobs will fail.

**Context**

This seems to have started with PR #1201.

**Detailed Description**

**Additional Information**

**Possible Implementation**

Write a function to swap the location of memxxx and atmos when copying data from the `icsdir` to the `comrot` OR if the data has already been staged.  I accomplished this with the following line, performed in `$COMROT/enkfgdas.YYYYMMDD/CC/`:
```
set -eu
cwd=$(pwd)
cd atmos
for dir in mem*; do
   mkdir $dir/atmos
   mv $dir/{INPUT,*txt} $dir/atmos
   mv $dir $cwd/
done
cd $cwd
rm -rf atmos
```

This function would need to be intelligent enough to know it should only execute on the first cycle.

Alternatively, this could be addressed in gdas_init.

**Comment by aerorahul** [comment]
@DavidHuber-NOAA 
I think this issue is best addressed in `gdas_init` to stage the files in the directory structure that the workflow expects.
We can work on it in the ufs-utils with an updated hash.

We can keep this issue open and link it to the issue in ufs-utils.

**Comment by DavidHuber-NOAA** [comment]
Alright, I've opened noaa-emc/ufs_utils#762.

**Comment by DavidHuber-NOAA** [comment]
UFS_utils was updated to include a fix at hash [93e85ace](https://github.com/ufs-community/UFS_UTILS/commit/93e85ace055a8294dc6f5d822fe2b489583d719b).
